backend/src/store/room.go

```diff
@@ -251,6 +272,17 @@ func (r *Room) SetAutoReveal(autoReveal bool) {
 	r.EmitState()
 }
 
+func (r *Room) SetTimebox(timeboxInSeconds uint) {
+	r.mtx.Lock()
+	defer r.mtx.Unlock()
+	r.timeboxInSeconds = timeboxInSeconds
+	log.WithFields(log.Fields{
+		"timebox": r.timeboxInSeconds,
+		"roomID":  r.id,
+	}).Info("set to auto reveal after the timebox is seconds has run out")
```

Comment:
I don't quite get this message.
I think I would not mention auto reveal, so that in the future we don't think that it has something to do with the auto-reveal feature.
Is there a reason why we explain the feature at every `SetTimebox`?
How about something along lines of: 
`"set timebox seconds"`